packages/trader/src/Modules/Account/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -0,0 +1,169 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { DesktopWrapper, MobileWrapper } from '@deriv/components';
+import { convertDateFormat } from '@deriv/shared/utils/date';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import DataTable from 'App/Components/Elements/DataTable';
+import { WS } from 'Services/ws-methods';
+import Loading from '../../../../../templates/app/components/loading.jsx';
+import LoadErrorMessage from '../../ErrorMessages/LoadErrorMessage';
+
+const Td = ({ title, text, className }) => (
+    <td>
+        <h3 className='cell-title'>{title}</h3>
+        <span className={classNames('cell-value', className)}>{text}</span>
+    </td>
+);
+
+const LoginHistoryListView = ({ fields, login_history }) => (
+    <table className='login-history-table'>
+        <tbody>
+            <tr>
+                <Td title={fields.date} text={login_history.date} />
+            </tr>
+            <tr>
+                <Td title={fields.browser} text={login_history.browser} />
+                <Td className='action' title={fields.action} text={login_history.action} />
+            </tr>
+            <tr>
+                <Td title={fields.ip} text={login_history.ip} />
+                <Td title={fields.status} text={login_history.status} />
+            </tr>
+        </tbody>
+    </table>
+);
+
+const parseUA = user_agent => {
+    const lookup = [
+        { name: 'Edge', regex: /(?:edge|edga|edgios|edg)\/([\d\w.-]+)/i },
+        { name: 'SeaMonkey', regex: /seamonkey\/([\d\w.-]+)/i },
+        { name: 'Opera', regex: /(?:opera|opr)\/([\d\w.-]+)/i },
+        { name: 'Chromium', regex: /(?:chromium|crios)\/([\d\w.-]+)/i },
+        { name: 'Chrome', regex: /chrome\/([\d\w.-]+)/i },
+        { name: 'Safari', regex: /version\/([\d\w.-]+)/i },
+        { name: 'IE', regex: /msie\s([\d.]+[\d])/i },
+        { name: 'IE', regex: /trident\/\d+\.\d+;.*[rv:]+(\d+\.\d)/i },
+        { name: 'Firefox', regex: /firefox\/([\d\w.-]+)/i },
+        { name: 'Binary app', regex: /binary\.com V([\d.]+)/i },
+    ];
+    for (let i = 0; i < lookup.length; i++) {
+        const info = lookup[i];
+        const match = user_agent.match(info.regex);
+        if (match !== null) {
+            return {
+                name: info.name,
+                version: match[1],
+            };
+        }
+    }
+    return null;
+};
+
+class LoginHistory extends React.Component {
+    // data keys+values
+    fields = {
+        date: localize('Date and time'),
+        action: localize('Action'),
+        browser: localize('Browser'),
+        ip: localize('IP address'),
+        status: localize('Status'),
+    };
+    // state
```

Comment:
nit: is this comment relevant?

Reply:
I have the habit of separating state from class methods. If we don't need them then I can remove. Please let me know the practice in this case.

Reply:
I mean the comment doesn't really add any value as state is declared right below it :)

Reply:
Fair enough :+1: